lib/msal-core/src/utils/WindowUtils.ts

```diff
@@ -174,13 +185,13 @@ export class WindowUtils {
      * @ignore
      */
     static getPopUpWithHash(hash: string): Window {
-        return WindowUtils.getPopups().find(popup => {
+        return WindowUtils.getPopups().filter(popup => {
```

Comment:
does this always return an array? Could it be null?

Reply:
Get popups will always return an array, yes.

Reply:
As will filter.